crates/cheatcodes/src/evm/mock.rs

```diff
@@ -102,10 +121,25 @@ fn mock_call(
     value: Option<&U256>,
     rdata: &Bytes,
     ret_type: InstructionResult,
+) {
+    mock_calls(state, callee, cdata, value, &[rdata.clone()], ret_type)
+}
+
+#[allow(clippy::ptr_arg)] // Not public API, doesn't matter
```

Comment:
these are no longer needed

Reply:
Removed from both `mock_call` and `mock_calls`